cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -11647,6 +11844,464 @@ ir.cpp:
+# 2156|     mu2156_3(int)       = Store[x]           : &:r2156_1, r2156_2
+#-----|   Goto -> Block 3
+
+# 2158|   Block 2
+# 2158|     r2158_1(glval<int>) = VariableAddress[y] : 
+# 2158|     r2158_2(int)        = Constant[0]        : 
+# 2158|     mu2158_3(int)       = Store[y]           : &:r2158_1, r2158_2
+#-----|   Goto -> Block 3
+
+# 2159|   Block 3
+# 2159|     r2159_1(glval<String>)  = VariableAddress[s]              : 
+# 2159|     r2159_2(glval<unknown>) = FunctionAddress[~String]        : 
+# 2159|     v2159_3(void)           = Call[~String]                   : func:r2159_2, this:r2159_1
+# 2159|     mu2159_4(unknown)       = ^CallSideEffect                 : ~m?
+# 2159|     v2159_5(void)           = ^IndirectReadSideEffect[-1]     : &:r2159_1, ~m?
+# 2159|     mu2159_6(String)        = ^IndirectMayWriteSideEffect[-1] : &:r2159_1
+# 2160|     v2160_1(void)           = NoOp                            : 
+# 2154|     v2154_6(void)           = ReturnVoid                      : 
+# 2154|     v2154_7(void)           = AliasedUse                      : ~m?
+# 2154|     v2154_8(void)           = ExitFunction                    : 
+
+# 2169| void IfDestructors3(bool)
+# 2169|   Block 0
+# 2169|     v2169_1(void)           = EnterFunction                   : 
+# 2169|     mu2169_2(unknown)       = AliasedDefinition               : 
+# 2169|     mu2169_3(unknown)       = InitializeNonLocal              : 
+# 2169|     r2169_4(glval<bool>)    = VariableAddress[b]              : 
+# 2169|     mu2169_5(bool)          = InitializeParameter[b]          : &:r2169_4
+# 2170|     r2170_1(glval<Bool>)    = VariableAddress[B]              : 
+# 2170|     mu2170_2(Bool)          = Uninitialized[B]                : &:r2170_1
+# 2170|     r2170_3(glval<unknown>) = FunctionAddress[Bool]           : 
+# 2170|     r2170_4(glval<bool>)    = VariableAddress[b]              : 
+# 2170|     r2170_5(bool)           = Load[b]                         : &:r2170_4, ~m?
+# 2170|     v2170_6(void)           = Call[Bool]                      : func:r2170_3, this:r2170_1, 0:r2170_5
+# 2170|     mu2170_7(unknown)       = ^CallSideEffect                 : ~m?
+# 2170|     mu2170_8(Bool)          = ^IndirectMayWriteSideEffect[-1] : &:r2170_1
+
+# 2170|   (no string representation)
+# 2170|     CopyValue: (condition decl)
+# 2170|     ConditionalBranch: (condition decl)
+#-----|   False -> Block 3
+#-----|   True -> Block 2
+
+# 2170|   Block 1
+# 2170|     r2170_9(glval<Bool>)     = VariableAddress[B]              : 
+# 2170|     r2170_10(glval<unknown>) = FunctionAddress[operator bool]  : 
+# 2170|     r2170_11(bool)           = Call[operator bool]             : func:r2170_10, this:r2170_9
+# 2170|     mu2170_12(unknown)       = ^CallSideEffect                 : ~m?
+# 2170|     v2170_13(void)           = ^IndirectReadSideEffect[-1]     : &:r2170_9, ~m?
+# 2170|     mu2170_14(Bool)          = ^IndirectMayWriteSideEffect[-1] : &:r2170_9
+
```

Comment:
Hm. Do you know what's going on here? It seems like:
1. `Block 0` ends without any outgoing edges
2. There's a block named `(no string representation)` that looks super strange.
3. `Block 1` also has no outgoing edges

Is this related to this PR? Or to variable declarations inside if expressions?